.github/scripts/fxcore-sync-up-version.js

```diff
@@ -0,0 +1,24 @@
+const path = require("path");
+const semver = require("semver");
+const fse = require("fs-extra");
+
+const templatePath = path.join(__dirname, "../../templates")
+const templateVersion = require(path.join(templatePath, "package.json")).version
+const templateName = require(path.join(templatePath, "package.json")).version
```

Comment:
templateName is same with templateVersion?

Reply:
sorry.